rules/linux/macos_remote_system_discovery.yml

```diff
@@ -0,0 +1,27 @@
+title: Macos Remote System Discovery
+id: 11063ec2-de63-4153-935e-b1a8b9e616f1
+status: experimental
+description: Detects the enumeration of other remote systems.
+author: Alejandro Ortuno, oscd.community
+date: 2020/10/22
+references:
+  - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1018/T1018.md
+logsource:
+  category: process_creation
+  product: macos
+detection:
+  selection_1:
+    ProcessName|endswith:
+      - '/arp'
+    CommandLine|contains|all:
+      - '-a'
+  selection_2:
+    ProcessName|endswith:
+      - '/ping'
```

Comment:
we need to consider adding some filters here, as it gonna be extremely noisy... at least exclude public IP ranges